<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtlHomPage</class>
 <widget class="QDialog" name="HtlHomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_top">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hotel Home Page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>291</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="EditInfoBtn">
      <property name="text">
       <string>Edit Info</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CheckRoomBtn">
      <property name="text">
       <string>Check Room</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CheckOrderBtn">
      <property name="text">
       <string>Check Order</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddRoomBtn">
      <property name="text">
       <string>Add Room</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LogoutBtn">
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="TodayLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Today is</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="d_TodayLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
